src/redux/data.js

```diff
@@ -809,13 +809,15 @@ export const dataAddNewTransaction = (
         type: DATA_ADD_NEW_TRANSACTION_SUCCESS,
       });
       saveLocalTransactions(_transactions, accountAddress, network);
-      await dispatch(
-        incrementNonce(
-          parsedTransaction.from,
-          parsedTransaction.nonce,
-          parsedTransaction.network
-        )
-      );
+      if (parsedTransaction.from && parsedTransaction.nonce) {
```

Comment:
the error was swallowed on line 839, resulting in undefined values in the result